src/debugger/gdb-stub.c

```diff
@@ -243,6 +314,29 @@ static void _readGPRs(struct GDBStub* stub, const char* message) {
 	_sendMessage(stub);
 }
 
+static void _writeRegister(struct GDBStub* stub, const char* message) {
+	struct ARMCore* cpu = stub->d.core->cpu;
+	const char* readAddress = message;
+
+	unsigned i = 0;
+	uint32_t reg = _readHex(readAddress, &i);
+	readAddress += i + 1;
+	uint32_t value = __builtin_bswap32(_readHex(readAddress, &i));
```

Comment:
This isn't compatible with MSVC. Please ifdef this and use _byteswap_ulong for MSVC.